options-table.c

```diff
@@ -981,6 +981,24 @@ const struct options_table_entry options_table[] = {
 	  .text = "The default colour palette for colours zero to 255."
 	},
 
+	{ .name = "popup-style",
+	  .type = OPTIONS_TABLE_STRING,
+	  .scope = OPTIONS_TABLE_WINDOW,
```

Comment:
Seems like the `popup-style` and `popup-border-style` options are already documented in the "Available window option" section, so it seems this can stay as is :)